charts/datadog/README.md

```diff
@@ -608,6 +608,8 @@ helm install <RELEASE_NAME> \
 | clusterAgent.image.pullSecrets | list | `[]` | Cluster Agent repository pullSecret (ex: specify docker registry credentials) |
 | clusterAgent.image.repository | string | `nil` | Override default registry + image.name for Cluster Agent |
 | clusterAgent.image.tag | string | `"7.59.0"` | Cluster Agent image tag to use |
+| clusterAgent.kubernetesApiserverCheck | object | `{"disableUseComponentStatus":false}` | correspond to options for configuring the kube_apiserver integration. |
```

Comment:
this line should not be part of the readme

Reply:
@clamoriniere thanks it's fixed.